### ISSUE TITLE
literalMidWordAsterisks and single character bolding

### ISSUE THREAD

**Issue by ctsears** [showdownjs/showdown#478]
With the literalMidWordAsterisks option enabled, the following markdown doesn't render properly..

```
Click the **X** button
```

Instead of the X character being bolded, it appears in italics with wrapping asterisks.

If there is subsequent bolding, it also renders incorrectly, though not in the same fashion. 

```
Click the **X** button and then the **OK** button
```

**Comment by tivie** [comment]
Verified. Introduced in [1.8.3](https://github.com/showdownjs/showdown/compare/1.8.2...1.8.3)

**Comment by tivie** [comment]
Should be fixed now in fe70e45

> ## Donate [![Click here to lend your support to: ShowdownJS website and testing platform and make a donation at pledgie.com](https://pledgie.com/campaigns/35166.png?skin_name=chrome)](https://pledgie.com/campaigns/35166)
>As you know, ShowdownJS is a free library and it will remain free forever. However, maintaining and improving the library costs time and money.
>If you like our work and find our library useful, please donate [through Pledgie](https://pledgie.com/campaigns/35166) or directly [through paypal](https://www.paypal.me/tiviesantos)!! Your contribution will be greatly appreciated and help us continue to develop this awesome library.

**Comment by ctsears** [comment]
That fixed it. Thanks!
